Store parallel magnetic field

dschwoerer opened the pull request.

dschwoerer committed:
store parallel magnetic field

ZedThree reviewed (changes requested):
Seeing as both places that call `get_metric` put `B` into `metric["Bxy"], metric["B"]`, it would be better to move this into `get_metric` (and not return `B`)

dschwoerer committed:
Add everything we want to store directly into metric

This also adds pressure. While it may not be used now
it will probably be useful to have the non-interpolated
values once it is used.

dschwoerer commented:
Good point!
I also added pressure, as we may want to use that in the future, and it makes the code easier and more consistent.
